engine/ladon/doc.go

```diff
@@ -245,6 +245,11 @@ type listOryAccessControlPolicyRoles struct {
 	// required: true
 	Flavor string `json:"flavor"`
 
+	// The Member (ID) for which the roles are to be listed (Optional).
```

Comment:
I tried adding Member []string in Roles and Sub, Action, Res in Policies but swagger generate won't add it in the spec. Can you share your process for adding URL query params in the API in the swagger doc

Reply:
Since this is `in: query` it can only be a raw string _I think_ (not sure though).

Reply:
Even that doesn't work

Reply:
Hm, maybe I misunderstood the problem? Are you trying for this to become a `[]string`?

Reply:
The problem is that swagger spec generation is not adding any new field I add to these request structs. I tried both Member string and Member []string but after running make sdk the swagger query params for list only has offset and limit.

Reply:
Ok I will check it out now

Reply:
Okay, finally I was able to solve that issue. The brew installed version of go-swagger had some caching issue and wouldn't generate new swagger spec on changes. I had to install go-swagger from source and then it worked as expected. Phewww!!!